### PR TITLE
appium: fix linkage audit on Linux

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/appium.rb [hunk 1]
```diff
@@ -18,7 +18,9 @@ class Appium < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    disable_chromedriver = ""
+    on_linux { disable_chromedriver << "--chromedriver-skip-install" }
```
Comment (SMillerDev) [inline Formula/appium.rb:22]: We probably want this regardless since IIRC it's prety much downloading an unversioned Chrome driver.

Comment (danielnachun) [reply Formula/appium.rb:22]: That makes sense, and I think it's even possible for a user to download and install the Chrome driver as a separate package, so it's not a major inconvenience to disable everywhere.

**Comment by danielnachun** [comment]
Although disabling the Chrome driver fixes the issues with that unversioned binary, we've stumbled upon the  unversioned `instruments-without-delay` binaries on macOS now.  There is a Makefile for building the binaries: https://github.com/appium/appium-ios-driver/blob/master/Makefile.  

However, the GitHub repo it links to (https://github.com/facebookarchive/instruments-without-delay) has no tagged releases and says that it is not being maintained.  I suppose we could take the latest revision (from 2015) as our "version", but this is definitely bending our rules quite a bit.  On the other hand, we've discussed before our desire not to break formuale that people are using, and this one had >7500 downloads in the last year.  What are our thoughts on the best way forward?

**Comment by SMillerDev** [comment]
Does it pass tests without that binary? We can just try removing it, I'm sure my colleagues will let me know if that breaks things 😅 .

**Comment by danielnachun** [comment]
I took a closer look at `instruments-without-delay` and it seems the last version it officially supported was iOS 8 👎.  In fact the entire Node module which pulls in `instruments-without-delay` is obsolete becuase it only supports up to iOS 9.3: https://github.com/appium/appium-ios-driver.  Its functionality has now been superseded by https://github.com/appium/appium-xcuitest-driver, and given that iOS 9 was released in 2015 I suspect nobody has used `appium-ios-driver` for some time.  I have deleted the entire folder for `appium-ios-driver` and it passed the test with no issues.  I added a comment as well to explain why the folder is being deleted.

**Review by SMillerDev — DISMISSED** [review]
Thanks @danielnachun!

**Comment by danielnachun** [comment]
I had to push one more change because I accidentally left in something in the commit I was testing earlier.  Can I get this approved again?  Thanks!

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**